lib/generators/koi/admin_views/admin_views_generator.rb

```diff
@@ -79,7 +79,7 @@ def index_attribute_for(attribute)
       when :rich_text
         %(<% row.rich_text :#{attribute.name} %>)
       when :attachment
-        %(<% row.image :#{attribute.name} %>)
+        %(<% row.attachment :#{attribute.name} %>)
```

Comment:
I don't understand why this is desirable. Wouldn't we like to see a preview of the image if one is available?

Rails' active storage guide recommends using `representable?` to decide whether to render a preview or a download link.

Reply:
On the generator, since we are not aware of the type of `attachment`, felt generating an image component was incorrect. 

I'll update the Body::AttachmentComponent to show a preview of the attachment if it is `representable?`